
sensor_msgs/Image[] image 
sensor_msgs/CameraInfo[] camera_info

# this is the update for each of the particles
# this is applied to each particle before reevaluating
geometry_msgs/Twist twist
---

geometry_msgs/Pose pose
float64 weight
int32 num_particles 